lambda/config.py

```diff
@@ -0,0 +1,77 @@
+import boto3
+from botocore.exceptions import ClientError
+
+from logger import LOG
+
+
+def get_ssm_param(param: str) -> str:
```

Comment:
Should this raise an error if the parameter can't be found? Can the calling function work correctly with `None` instead of a valid parameter/token/password?

Reply:
The documentation should state that a null check is needed before using the return value.